backends/platform/3ds/osystem-graphics.cpp

```diff
@@ -239,12 +388,12 @@ void OSystem_3DS::grabPalette(byte *colors, uint start, uint num) const {
 }
 
 void OSystem_3DS::copyRectToScreen(const void *buf, int pitch, int x,
-                                           int y, int w, int h) {
+                                   int y, int w, int h) {
 	Common::Rect rect(x, y, x+w, y+h);
 	_gameScreen.copyRectToSurface(buf, pitch, x, y, w, h);
 	Graphics::Surface subSurface = _gameScreen.getSubArea(rect);
 
-	Graphics::Surface *convertedSubSurface = subSurface.convertTo(_pfGameTexture, _palette);
+	Graphics::Surface *convertedSubSurface = subSurface.convertTo(_gfxState.gfxMode->svmBufferPF, _palette);
```

Comment:
Wouldn't the code be clearer if it said we are converting to the pixel format of `_gameTopTexture`?